src/middle-end/auth/selectors.js

```diff
@@ -21,3 +21,20 @@ exports.getIsAuthenticated = ({ auth }) => {
 
     return !!(index.result && !index.error);
 };
+
+exports.getToken = ({ auth }) => {
+
+    const { [LOGIN.BASE]: index } = auth.indexes;
+    const { token } = index.result || {};
+
+    if (typeof token !== 'undefined') {
+
+        if (!token) {
```

Comment:
This looks like code that might be supposed to live in the login action.  For example, when you first visit the site (and you don't have a token in storage), we want this action to fail immediately with an error like the one here: https://github.com/BigRoomStudios/strangeluv/pull/241/files#diff-dfbba6a3d45354bb54b3cefa957b7eb1R31